app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+  def new
+  end
+
+  def create
+    @user = User.find_by_email(params[:email])
+    if @user.authenticate(params[:password])
+      session[:user_id] = @user.id
+      flash[:success] = "Successfully logged in!"
+      redirect_to dashboard_path
+    end
```

Comment:
@drod1000 I think we should add an error message here flash[:danger] = "Invalid login" or some other more detailed message to cover the negative case. Otherwise the SessionController looks great! FYI @Dpalazzari @njgheorghita